website/content/docs/connect/registration/sidecar-service.mdx

```diff
@@ -1,12 +1,11 @@
 ---
 layout: docs
-page_title: Connect - Sidecar Service Registration
-description: |-
-  Sidecar service registrations provide a convenient shorthand for registering a
-  sidecar proxy inline with a regular service definition.
+page_title: Service Mesh Proxy: Sidecar Configuration
```

Comment:
I don't like how clunky the title I suggested is, but it's a more accurate reflection of what is going on (and how it's different from the alternative).
```suggestion
page_title: Service Mesh Proxy Registration: Included in Service Registration
```